core/src/broadcast_stage/standard_broadcast_run.rs

```diff
@@ -418,6 +469,8 @@ impl BroadcastRun for StandardBroadcastRun {
         blockstore_sender: &Sender<(Arc<Vec<Shred>>, Option<BroadcastShredBatchInfo>)>,
     ) -> Result<()> {
         let receive_results = broadcast_utils::recv_slot_entries(receiver)?;
+        // TODO: Confirm that last chunk of coding shreds
```

Comment:
A good way to confirm this might be by monitoring the `broadcast-transmit-shreds-stats` metric, `end_to_end_elapsed` field. It doesn't look like it should be too big of a deal because even with these changes, it looks like at most we will have at most an additional `MAX_DATA_SHREDS_PER_FEC_BLOCK` at the last iteration of `process_receive_results` for this slot compared to the code today